open-metadata-resources/open-metadata-archives/content-pack-helpers/src/main/java/org/odpi/openmetadata/samples/archiveutilities/SimpleCatalogArchiveHelper.java

```diff
@@ -5229,4 +5229,31 @@ public void addReferenceDataClassification(String assetGUID)
 
         archiveBuilder.addClassification(archiveHelper.getClassificationEntityExtension(referenceableEntityProxy, classification));
     }
+
+    /**
+     * Add a SecurityTags classification to the requested element.
+     *
+     * @param assetGUID unique identifier for the element to classify
+     * @param securityLabels list of security labels
+     * @param securityProperties map of security properties
+     */
+    public void addSecurityTagsClassification(String              assetGUID,
+                                              List<String>        securityLabels,
+                                              Map<String, Object> securityProperties)
```

Comment:
Is it possible to add support for the third attribute called `accessGroups`?  This is a new property added in 4.0. See https://egeria-project.org/types/4/0423-Security-Definitions/